virtcontainers/container.go

```diff
@@ -984,13 +984,19 @@ func (c *Container) update(resources specs.LinuxResources) error {
 
 	newResources := ContainerResources{
 		VCPUs: uint32(utils.ConstraintsToVCPUs(*resources.CPU.Quota, *resources.CPU.Period)),
-		MemMB: uint32(*resources.Memory.Limit >> 20),
+		// do page align to memory, as cgroup memory.limit_in_bytes will be aligned to page when effect.
+		// TODO use GetGuestDetails to get the guest OS page size.
+		MemByte: (*resources.Memory.Limit >> 12) << 12,
```

Comment:
This is still assuming 4KB page size. Please use `os.Getpagesize()` for it.

Reply:
Is this comment still valid and if so does it still need to be addressed?

Reply:
Yes, the comment and TODO are still valid, I will create a issue for this TODO.